### PR TITLE
find_storage_pool_sources: Add cases for local host and no srcSpec test

### PR DESCRIPTION
**Pull request by hahakiki2010**
Signed-off-by: Yan Li <yannli@redhat.com>

### CODE REVIEW

**Comment by hahakiki2010** [comment]
Test passed.
(virtenv_py3) [root@localhost tp-libvirt]# avocado run --vt-type libvirt virsh.find_storage_pool_sources
JOB ID     : c5a61e01ceb3288b13658fc1578d331a26a1a741
JOB LOG    : /root/avocado/job-results/job-2018-07-04T17.59-c5a61e0/job.log
 (01/12) type_specific.io-github-autotest-libvirt.virsh.find_storage_pool_sources.positive_test.non_acl.local_host.no_srcSpec: PASS (28.57 s)
 (02/12) type_specific.io-github-autotest-libvirt.virsh.find_storage_pool_sources.positive_test.non_acl.remote_host.nfs_source.specified_srcSpec: PASS (2.54 s)
 (03/12) type_specific.io-github-autotest-libvirt.virsh.find_storage_pool_sources.positive_test.non_acl.remote_host.iscsi_source.specified_srcSpec: PASS (27.01 s)
 (04/12) type_specific.io-github-autotest-libvirt.virsh.find_storage_pool_sources.positive_test.non_acl.remote_host.logical_source.specified_srcSpec: PASS (28.17 s)
 (05/12) type_specific.io-github-autotest-libvirt.virsh.find_storage_pool_sources.positive_test.acl_test.local_host.no_srcSpec: PASS (30.33 s)
 (06/12) type_specific.io-github-autotest-libvirt.virsh.find_storage_pool_sources.positive_test.acl_test.remote_host.nfs_source.specified_srcSpec: PASS (4.33 s)
 (07/12) type_specific.io-github-autotest-libvirt.virsh.find_storage_pool_sources.positive_test.acl_test.remote_host.iscsi_source.specified_srcSpec: PASS (28.84 s)
 (08/12) type_specific.io-github-autotest-libvirt.virsh.find_storage_pool_sources.positive_test.acl_test.remote_host.logical_source.specified_srcSpec: PASS (30.32 s)
 (09/12) type_specific.io-github-autotest-libvirt.virsh.find_storage_pool_sources.negative_test.invalid_type: PASS (1.49 s)
 (10/12) type_specific.io-github-autotest-libvirt.virsh.find_storage_pool_sources.negative_test.invalid_srcSpec: PASS (2.41 s)
 (11/12) type_specific.io-github-autotest-libvirt.virsh.find_storage_pool_sources.negative_test.readonly_test: PASS (28.66 s)
 (12/12) type_specific.io-github-autotest-libvirt.virsh.find_storage_pool_sources.negative_test.acl_test: PASS (4.88 s)
RESULTS    : PASS 12 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 381.72 s

**Comment by iccaszhulili** [comment]
rh-caselink retest please

File: libvirt/tests/cfg/virsh_cmd/pool/virsh_find_storage_pool_sources.cfg [hunk 1]
```diff
@@ -11,13 +11,25 @@
...
+                    only specified_srcSpec
+                    source_host = "127.0.0.1"
+                    variants:
+                        - nfs_source:
+                            source_type = "netfs"
+                        - iscsi_source:
+                            source_type = "iscsi"
+                        - logical_source:
+                            source_type = "logical"
+                            vg_name = "virttest_vg_0"
```
Comment (kylazhang) [inline libvirt/tests/cfg/virsh_cmd/pool/virsh_find_storage_pool_sources.cfg]: For previous cases, source_Spec = "" will test with different type of source, but after you change, it only test local host, is it expected?

Comment (hahakiki2010) [reply libvirt/tests/cfg/virsh_cmd/pool/virsh_find_storage_pool_sources.cfg]: In previous cases, no cases for src_Spec = "", it must be specified, either a Invalid xml or a Valid xml. You can check the difference below.

**Pull request merged**